libm/riscv64/fenv.c

```diff
@@ -95,8 +95,16 @@ int fegetround(void)
 
 int fesetround(int round)
 {
-  asm volatile ("fsrm %z0" : : "r" (round));
-  return 0;
+  switch (round) {
```

Comment:
suggest use if-else instead of switch-cases
and please add some comment that we have not support RMM for riscv for memo

Reply:
> suggest use if-else instead of switch-cases and please add some comment that we have not support RMM for riscv for memo

Okay.